### PR TITLE
Update shell script

### PR DESCRIPTION
**Pull request by bbaayezi**
There is no need to install postgres on the local machine, you can just execute the `psql` command inside the postgres container and open an interactive shell using `docker exec -it` command.

### CODE REVIEW

**Review by lukasotocerny — APPROVED** [review]
LGTM 👍

**Comment by bbaayezi** [comment]
There may be further changes made to the shell scripts (change DB schema), so I will leave this PR open

File: CreateDB.sql [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
-CREATE TABLE feedback(
-    feedback_id varchar(36),
+CREATE TABLE IF NOT EXISTS projects(
     project_id varchar(22),
-    user_id varchar(36),
+    primary key (project_id)
+);
+
+CREATE TABLE IF NOT EXISTS feedbacks(
+    feedback_id uuid,
```
Comment (lukasotocerny) [inline CreateDB.sql]: I'd add default UUID generation, `feedback_ui uuid NOT NULL DEFAULT gen_random_uuid()` with the extension `pgcrypto` as described [here](https://dba.stackexchange.com/a/223124). Also I'd explicitly add `NOT NULL`, to make it more obvious.

File: CreateDB.sql [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
+    primary key (feedback_id),
+    foreign key(project_id) references projects(project_id)
 );
 
-CREATE TABLE votes(
-    vote_id varchar(36),
-    feedback_id varchar(36),
-    user_id varchar(36),
+CREATE TABLE IF NOT EXISTS votes(
+    vote_id uuid,
```
Comment (lukasotocerny) [inline CreateDB.sql]: Same as previous comment

File: CreateDB.sql [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
+    v_timestamp bigint,
     primary key (vote_id),
-    foreign key (feedback_id) references feedback(feedback_id)
+    foreign key (feedback_id) references feedbacks(feedback_id)
 );
 
-CREATE TABLE conversions(
-    conversion_id varchar(36),
+CREATE TABLE IF NOT EXISTS conversions(
+    conversion_id uuid,
```
Comment (lukasotocerny) [inline CreateDB.sql]: Same as previous comment

**Review by lukasotocerny — CHANGES_REQUESTED** [review]
I'd generate automatically UUIDs by default and state `NOT NULL` for better readability.

**Review by lukasotocerny — APPROVED** [review]
LGTM

**Pull request merged**